plugins/lighthouse/src/components/AuditList/AuditListForEntity.tsx

```diff
@@ -13,19 +13,65 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { AuditListTable } from './AuditListTable';
 import { useWebsiteForEntity } from '../../hooks/useWebsiteForEntity';
-import { Progress } from '@backstage/core-components';
+import { LIGHTHOUSE_WEBSITE_URL_ANNOTATION } from '../../../constants';
+import {
+  Content,
+  ContentHeader,
+  InfoCard,
+  Progress,
+  WarningPanel,
+} from '@backstage/core-components';
+import { Button } from '@material-ui/core';
+import { resolvePath, useNavigate } from 'react-router-dom';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { useRouteRef } from '@backstage/core-plugin-api';
+import { rootRouteRef } from '../../plugin';
+import LighthouseSupportButton from '../SupportButton';
 
 export const AuditListForEntity = () => {
   const { value, loading, error } = useWebsiteForEntity();
+  const { entity } = useEntity();
+  const navigate = useNavigate();
+  const fromPath = useRouteRef(rootRouteRef)?.() ?? '../';
+
+  let createAuditButtonUrl = 'create-audit';
+  const websiteUrl =
+    entity.metadata.annotations?.[LIGHTHOUSE_WEBSITE_URL_ANNOTATION] ?? '';
+  if (websiteUrl) {
+    createAuditButtonUrl += `?url=${encodeURIComponent(websiteUrl)}`;
+  }
+
+  let content: ReactNode = null;
+  content = (
+    <Content>
+      <ContentHeader title="Latest Audit">
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => navigate(resolvePath(createAuditButtonUrl, fromPath))}
+        >
+          Create New Audit
+        </Button>
+        <LighthouseSupportButton />
+      </ContentHeader>
+      <AuditListTable items={value ? [value] : []} />
+    </Content>
+  );
+
   if (loading) {
-    return <Progress />;
+    content = <Progress />;
   }
-  if (error || !value) {
-    return null;
+  if (error && !error.message.includes('no audited website found for url')) {
```

Comment:
I guess that we're not in control of this error message right? Does this come from lighthouse?

Reply:
Yes, this error  message comes from the `lighthouse-audit-service` api.